jsonnet/thanos-querier.libsonnet

```
@@ -418,6 +428,7 @@ function(params)
                   '-cookie-secret-file=/etc/proxy/secrets/session_secret',
                   '-openshift-ca=/etc/pki/tls/cert.pem',
                   '-openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt',
+                  '-bypass-auth-for=/-/(healthy|ready)',
```

Shouldn't we add beginning/end of text markers?
```suggestion
                  '-bypass-auth-for=^/-/(healthy|ready)$',
```